### DISCUSSION TITLE
How to use version 3.0.0?

### DISCUSSION THREAD

**Discussion by GodEnki** [lepoco/wpfui#562] [General]
Hi, first of all thank you for the great project. I see there is a lot of changes coming and I tried to use the preview nuget package (version 3.0.0 - preview.1). Is there any documentation how to use that? It seems that the template app from version 2.0.3 will no longer work?

**Comment by pomianowski** [comment]
Hey @GodEnki, version 3.0.0 is not yet released and there is no official documentation. If you want to see examples of how it works, you can build Wpf.Ui.Gallery.

If you want to make your own application with WPF UI 3.0+, you can use the pre-relase package by adding an entry to your csproj::
```xaml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>net7.0-windows10.0.22621.0</TargetFramework>
    <UseWPF>true</UseWPF>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="WPF-UI" Version="3.0.0-preview.1" />
  </ItemGroup>

</Project>

```

**Comment by GodEnki** [comment]
When I try to compile parts of the Wpf.Ui.Gallery, I get something like that: 
```
Error	XDG0008	The name "ImageIcon" does not exist in the namespace "http://schemas.lepo.co/wpfui/2022/xaml".
Error	XDG0008	The name "BreadcrumbBar" does not exist in the namespace "http://schemas.lepo.co/wpfui/2022/xaml".
Error	XDG0008	The name "SymbolIconSource" does not exist in the namespace "http://schemas.lepo.co/wpfui/2022/xaml".
``` 

so it seems that there are a few things missing in WPFUI 3.0?

**Reply by pomianowski** [reply]
> When I try to compile parts of the Wpf.Ui.Gallery, I get something like that:
> 
> ```
> Error	XDG0008	The name "ImageIcon" does not exist in the namespace "http://schemas.lepo.co/wpfui/2022/xaml".
> Error	XDG0008	The name "BreadcrumbBar" does not exist in the namespace "http://schemas.lepo.co/wpfui/2022/xaml".
> Error	XDG0008	The name "SymbolIconSource" does not exist in the namespace "http://schemas.lepo.co/wpfui/2022/xaml".
> ```
> 
> so it seems that there are a few things missing in WPFUI 3.0?

These controls are not yet published in NuGet, they will appear in **3.0.0-preview.2**

**Reply by GodEnki** [reply]
Thank you for the information! Then I'll wait for the next version!
